rmw_fastrtps_dynamic_cpp/src/rmw_node.cpp

```diff
@@ -57,6 +57,13 @@ rmw_destroy_node(rmw_node_t * node)
     eprosima_fastrtps_identifier, node);
 }
 
+rmw_ret_t
+rmw_node_assert_liveliness(const rmw_node_t * node)
+{
+  return rmw_fastrtps_shared_cpp::__rmw_node_assert_liveliness(
+    eprosima_fastrtps_identifier, node);
```

Comment:
Since we decided to keep the parameters vertical elsewhere, we should do that here as well.

Reply:
I thought it's only for functions whose signature can't fit 100 characters per line.